Userland/Libraries/LibSoftGPU/Device.cpp

```diff
@@ -113,6 +119,8 @@ static constexpr void setup_blend_factors(BlendFactor mode, FloatVector4& consta
 template<typename PS>
 static void rasterize_triangle(const RasterizerOptions& options, Gfx::Bitmap& render_target, DepthBuffer& depth_buffer, const Triangle& triangle, PS pixel_shader)
 {
+    g_num_rasterized_triangles++;
```

Comment:
Could we guard all these calculations with `if constexpr (ENABLE_STATISTICS_OVERLAY)` so they get culled during compilation when you do not enable this new mode?